src/components/AppInfo/AppInfo.jsx

```diff
@@ -16,20 +17,44 @@ import {
 } from 'react-bootstrap';
 
 import styles from './AppInfo.module.scss';
+import selector from './AppInfo.selector';
 
 import appIcon from '../../assets/app-icon.svg';
 import googleIcon from '../../assets/google-avatar.png';
 import facebookIcon from '../../assets/facebook-avatar.png';
+import { useEffect } from 'react';
```

Comment:
n this file there is already an import from the `react` library, this import needs to be moved above